Helper/StoreUrl.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CustomGento\DefaultStoreCodeRemover\Helper;
+
+use Magento\Store\Api\Data\StoreInterface;
+use Magento\Store\Model\Store;
+use MageWorx\SeoBase\Helper\StoreUrl as MageworxStoreUrl;
+
+if (!class_exists(MageworxStoreUrl::class)) {
```

Comment:
As Simon pointed out, this is not necessary, as we will never come into this class, if the original one does not exist, right?
So let's remove it, I would say.